### PR TITLE
README: clarify the Linux package install instructions

### PR DESCRIPTION
**Pull request by kenyon**

### CODE REVIEW

**Comment by crazy-max** [comment]
Missing DCO: https://github.com/docker/buildx/pull/1940/checks?check_run_id=15055638670

Otherwise LGTM. PTAL @thaJeztah @dvdksn

**Review by dvdksn — APPROVED** [review]
LGTM

**Comment by kenyon** [comment]
Rebased and added signoff.

File: README.md [hunk 1]
```diff
@@ -71,8 +71,9 @@ for Windows and macOS.
 
 ## Linux packages
 
-Docker Linux packages also include Docker Buildx when installed using the
-[DEB or RPM packages](https://docs.docker.com/engine/install/).
+Docker Engine package repositories contain Docker Buildx packages when installed according to the
+[Docker Enginer install documentation](https://docs.docker.com/engine/install/). Install the
```
Comment (thaJeztah) [inline README.md:75]: Looks like there was a typo here; `s/Enginer/Engine/`

Comment (kenyon) [reply README.md:75]: :( Fixed in #1993

**Pull request merged**